.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    "configurations": [
+        {
+            "name": "run audacity",
+            "type": "cppvsdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build/install/bin/audacity.exe",
+            "preLaunchTask": "build audacity",
```

Comment:
Probably there should be not a build here, but an install.
@kryksyh  suggested that we can just do `CMake: Install`, and not add a task file.

Reply:
> Probably there should be not a build here, but an install
This works because of the post-build task in the cmakelists that gets the executable copied to the install dir.
Having `install` as pre-launch task has for consequence that each time you launch after having modified the source code, the install target gets executed and, even if it's not the first time, just the check that everything (except audacity.exe) is up-to-date takes at least a couple of seconds.
On the other hand, for fast onboarding, what you suggest would be easier: the user wouldn't have to build the install target manually once.
But we can propose two launch tasks: the one you suggest and make it the default one, and for power users, the present one.

Reply:
> even if it's not the first time, just the check that everything (except audacity.exe) is up-to-date takes at least a couple of seconds

I'm confused. Now it's super fast for me even with install. Ok for your proposal, then.